### PR TITLE
Update workflow cucushift-installer-rehearse-aws-ipi-proxy-sdn-worker…

### PR DESCRIPTION
**Pull request by xiaojiey**
…s-rhel8 to enable fips on rhel nodes

### CODE REVIEW

**Comment by xiaojiey** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-debug

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -812,7 +812,7 @@ tests:
       TEST_ADDITIONAL: Network_Observability
     test:
     - chain: openshift-e2e-test-qe-destructive
-    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips
```
Comment (zhaozhanqi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:815]: I guess we need to update the job name with fips as well if the workflow is updated, right?

Comment (xiaojiey) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:815]: Good point. Seems E2E_RUN_TAGS also need to be updated.

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-workflow.metadata.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"path": "cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-workflow.yaml",
+	"path": "cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-workflow.metadata.json:2]: I think we should create a dedicated folder `fips` under `workers-rhel8` to be more clear.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml (hunk 1 above)
Comment (zhaozhanqi) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:815]: looks good to me

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 chain:
-  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-provision
+  as: cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision
```
Comment (yunjiang29) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml:2]: To be more accurate, if this `fips` present in chain name,  we should add `FIPS_ENABLED: "true"` to the chain (although it's already set in job config)

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml [hunk 4]
```diff
@@ -1,9 +1,10 @@
...
   steps:
   - chain: cucushift-installer-rehearse-aws-ipi-proxy-sdn-provision
   - ref: workers-rhel-aws-provision
   - ref: workers-rhel-pre-hook-sshtunnel
+  - ref: workers-rhel-pre-hook-fips
   - chain: workers-rhel
   env:
   - name: PLATFORM_VERSION
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml:10]: This provision chain is dedicated for fips testing, so suggest to move `FIPS_ENABLED` env to the provision chain, but not set it on job config level.

File: ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml (hunk 3 above)
Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/proxy/sdn/workers-rhel8-fips/provision/cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips-provision-chain.yaml:2]: Ah, the same comment as https://github.com/openshift/release/pull/44811/files#r1371436518, and I think we should remove `FIPS_ENABLED` from job config.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 5]
```diff
@@ -799,20 +799,20 @@ tests:
...
+- as: aws-ipi-proxy-sdn-workers-rhel8-fips-f28-destructive
   cron: 12 14 27 * *
   steps:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @proxy and @network-openshiftsdn and
-        not @fips'
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @proxy and @network-openshiftsdn'
+      FIPS_ENABLED: "true"
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:810]: We do not need the setting any more here

Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:810]: The same on `aws-ipi-proxy-sdn-workers-rhel8-fips-f14`

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml [hunk 6]
```diff
@@ -42,13 +42,12 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
-      BASE_DOMAIN: qe.devcluster.openshift.com
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml:45]: We need keep this line, right?

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml [hunk 7]
```diff
@@ -11966,7 +11966,7 @@ periodics:
         secretName: result-aggregator
 - agent: kubernetes
   cluster: build03
-  cron: 12 14 27 * *
```
Comment (jianlinliu) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:11969]: I saw there is no such change in job config, why here it is.

File: core-services/ci-chat-bot/workflows-config.yaml [hunk 8]
```diff
@@ -117,7 +117,7 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-proxy-techpreview:
     platform: aws
-  cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8:
+  cucushift-installer-rehearse-aws-ipi-proxy-sdn-workers-rhel8-fips:
```
Comment (jianlinliu) [inline core-services/ci-chat-bot/workflows-config.yaml:120]: I think we do not need to change it any more, if want a quick merge, I think we can revert the change on this file.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml (hunk 7 above)
Comment (xiaojiey) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:11969]: This is changed automatically when I run `make update`

Comment (jianlinliu) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml:11969]: got it.

**Comment by jianlinliu** [comment]
cc @liangxia to review
